lib/email/utils/helpers.js

```diff
@@ -28,7 +30,7 @@ function logFlowEventFromMessage(log, message, type) {
   const elapsedTime = currentTime - flowBeginTime
 
   if (flowId && flowBeginTime && (elapsedTime > 0) && type && templateName) {
-    const eventName = `email.${templateName}.${type}`
+    const eventName = `email.${templateName}.${type}.${emailDomain}`
```

Comment:
> Are we doing it this way because it's the right abstraction, or because it's the path of least resistance given our current metrics infrastructure?

Well it definitely wasn't the latter, but obviously that doesn't mean we ended up with the right abstraction. Maybe we just made the wrong decision or misunderstood something.

Fwiw my suggestion was to emit a separate `email.domain.${domain}` event because it seemed too niche a requirement to be a first-class column in `flow_events`. What do you think right abstraction is?